package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "js-yaml": "^3.13.1",
     "jsonc-parser": "^2.2.1",
+    "prettier": "2.0.5",
```

Comment:
IIRC this has to be an optional dependency, otherwise projects like Openshift Console cannot use the YAML Language Server because it generates a lot of webpack errors